datasets/nwm-archive.yaml

```diff
@@ -43,6 +45,22 @@ Tags:
   - sustainability
 License: Open Data. There are no restrictions on the use of this data.
 Resources:
+  - Description: The complete archive of NWM input forcing and model output data version 2.1 in NetCDF format.
+    ARN: arn:aws:s3:::noaa-nwm-retrospective-2-1-pds
+    Region: us-east-1
+    Type: S3 Bucket
+    '[AWS CLI] (https://aws.amazon.com/cli/)' Access (No AWS account required):
+    aws s3 ls s3://noaa-nwm-retrospective-2-1-pds/ --no-sign-request

```

Comment:
Thanks @Patrick-Keown. Looks like some extra stuff got added in here, `Type` should just be `S3 Bucket`. So if you remove lines 52-53 and again down at 60-61, I think this should pass.